### DISCUSSION TITLE
HDD sleep mode

### DISCUSSION THREAD

**Discussion by SebastianSava** [IceWhaleTech/CasaOS#1244] [Q&A]
Hello everyone!

I have a question regarding mounted drives for local-network sharing. Can the current implementation put drives to sleep? Like Windows does. I want to know if the units are active 24/7 even if they are not in use.

![image](https://github.com/IceWhaleTech/CasaOS/assets/67977875/fa8e1234-5c9d-42cc-baed-79415e3bb487)

I think this is a good approach for long-life drives.

**Comment by thedroidkid** [comment]
It should auto put the drive to sleep in Debian

**Comment by Sn0w3y** [comment]
You need to install hdparm with:

`sudo apt-get update && sudo apt-get install hdparm
`

After this set the Timer to spin down the HDD like:


` sudo hdparm -S 200 /dev/sda
`

This will set the Spindown to 16 Minutes + 40 seconds for example
